deno/fhirchat-demo/islands/SuggestionsIsland.tsx

```diff
@@ -0,0 +1,46 @@
+import { useContext } from "preact/hooks";
+import { AIConversationState } from "../hooks/aiConversationContext.ts";
+export default function SuggestionsIsland() {
+  const {
+    conversation,
+  } = useContext(AIConversationState);
+
+  return (
+    conversation.value.length === 0
+      ? (
```

Comment:
Nit: I think it's a bit easier to read if you put the negative case first, just so you don't need to scroll down to see it:
```suggestion
conversation.value.length > 0
      ? <></>
      : ...
```